gatling/PerformanceTest.scala

```diff
@@ -0,0 +1,103 @@
+package com.epam.gatling
+
+class PerformanceTest extends Simulation {
+
+  // URL
+  val user_url = "http://localhost:8090/api/v1"
+
+  //   Search transtation random
+  val transtationList = List("ru", "en", "de", "error")
+  val search_transtation_random_map = Iterator.continually(
+    Map("search_transtation_random" -> (transtationList(Random.nextInt(transtationList.size)))))
+
+  //   Search query random
+  val queryList = List("first", "second", "third", "fourth", "fifth", "sixth", "seventh", "last", "error")
+  val search_query_random_map = Iterator.continually(
+    Map("search_query_random" -> (queryList(Random.nextInt(queryList.size)))))
+
+  // Articles ID random
+  val articles_id_random_map = Iterator.continually(
+    Map("articles_id_random" -> Random.nextInt(250)))
+
+  // Categories ID random
+  val categories_id_random_map = Iterator.continually(
+    Map("categories_id_random" -> Random.nextInt(250)))
+
+  // Categories number random
+  val numberList = List("first", "second", "third", "fourth", "fifth", "sixth", "seventh", "last", "error")
+  val categories_number_random_map = Iterator.continually(
+    Map("categories_number_random" -> (numberList(Random.nextInt(numberList.size)))))
+
+  // Categories number random
+  val categories_transtation_random_map = Iterator.continually(
+    Map("categories_transtation_random" -> (transtationList(Random.nextInt(transtationList.size)))))
+
+  // SetUp SIMULATION:
+  val pause_time = 1
+  val scenario_repeats = 2
+  val setup_users_start_rate = 50
+  val setup_users_end_rate = 1500
+  val setup_duration1 = 1
+  val setup_duration5 = 5
+
+  val httpProtocol = http
+    .baseUrl(user_url)
+    .inferHtmlResources()
+    .acceptHeader("text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9")
+    .acceptEncodingHeader("gzip, deflate")
+    .acceptLanguageHeader("ru-RU,ru;q=0.9,en-US;q=0.8,en;q=0.7")
+    .userAgentHeader("Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/79.0.3945.79 Chrome/79.0.3945.79 Safari/537.36")
+
+  val headers_0 = Map(
+    "Proxy-Connection" -> "keep-alive",
+    "Upgrade-Insecure-Requests" -> "1")
+
+  val scn = scenario("AndreysTaskSimulation")
```

Comment:
Bad task name.
It's honour for me that test named after me, nevertheless, I prefer to give my name, for example, to the city or at least the street.
Scenario should contain the main goal of the scenario/